core/jvm/src/main/scala/cats/effect/unsafe/ScalQueue.scala

```diff
@@ -214,11 +386,96 @@ private[effect] final class ScalQueue[A <: AnyRef](threadCount: Int) {
    * Clears all concurrent queues that make up this Scal queue.
    */
   def clear(): Unit = {
-    val nq = numQueues
     var i = 0
-    while (i < nq) {
+    while (i < numQueues) {
       queues(i).clear()
+
+      // Reset all counters for this stripe
+      singletonsPresentCounts(i).set(0)
+      batchesPresentCounts(i).set(0)
+      fiberPresentCounts(i).set(0)
+
+      i += 1
+    }
+  }
+
+  /**
+   * Returns the total number of singleton tasks submitted to this queue.
+   */
+  def getTotalSingletonCount(): Long = {
+    var total = 0L
+    var i = 0
+    while (i < numQueues) {
+      total += singletonsSubmittedCounts(i).get()
```

Comment:
Let's make all the names consistent.
```suggestion
  def getTotalSingletonCount(): Long = {
    var total = 0L
    var i = 0
    while (i < numQueues) {
      total += totalSingletonCounts(i).get()
```

Reply:
I am sorry for that, I'll fix this.

Reply:
Hey @armanbilge , I was getting confused with the names. I will update it all, being consistent in the codebase.